packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-granules.js

```diff
@@ -574,3 +577,75 @@ test('writeGranules() does not persist records to Dynamo or Postgres if Postgres
     await t.context.granulePgModel.exists(knex, { granule_id: granuleId })
   );
 });
+
+test.serial('writeGranules() writes a granule and marks as failed if any file writes fail', async (t) => {
```

Comment:
These have to be run in serial because reasons. I'm guessing because a few of these tests directly update the `cumulusMessage`. I'll see if I can clean this up before merge.

Reply:
The `writeGranules` tests really do appear to need to be run in serial. We're writing, updating, and throwing errors based on database writes. I'm thinking running these in parallel causes collisions or unexpected database state and that's why I'm seeing intermittent failures when not run in `serial`.

Reply:
Interesting. All the tests were passing as non-serial before though, so the reason for the intermittent failures must be caused by this new test? I don't care a lot whether we run them in serial or not, but I do want to understand what test interdependencies we do/don't have